CONTRIBUTORS.md

```diff
@@ -15,3 +15,4 @@ SpeziML contributors
 
 * [Paul Schmiedmayer](https://github.com/PSchmiedmayer)
 * [Vishnu Ravi](https://github.com/vishnuravi)
+* [Philipp Zagar](https://github.com/philippzagar)
```

Comment:
🎉

Would be also great to add @AdritRao here as well so we keep the list up-to-date 👍